src/main/java/org/apache/ibatis/executor/keygen/Jdbc3KeyGenerator.java

```diff
@@ -63,9 +63,7 @@ public void processBatch(MappedStatement ms, Statement stmt, Object parameter) {
     if (keyProperties == null || keyProperties.length == 0) {
       return;
     }
-    ResultSet rs = null;
-    try {
-      rs = stmt.getGeneratedKeys();
+    try (final ResultSet rs = stmt.getGeneratedKeys()) {
```

Comment:
In this case, There is no need to explicitly specify the `final`.

Reply:
Thanks for reviewing, I have modified according to your comment